packages/devtools_app/lib/src/analytics/_analytics_web.dart

```diff
@@ -394,6 +394,14 @@ void timeEnd(
   );
 }
 
+void cancelTimingOperation(String screenName, String timedOperation) {
+  final operationKey = _operationKey(
+    screenName,
+    timedOperation,
+  );
+  _timedOperationsInProgress.remove(operationKey);
```

Comment:
add assert that the timing operation is actually in progress.

Reply:
done